src/Acuminator/Acuminator.Analyzers/Resources.resx

```diff
@@ -854,6 +854,15 @@ Reason: {2}</value>
   <data name="PX1102Title" xml:space="preserve">
     <value>The name of the additional delegate parameter for calling the base method of the PXOverride method breaks Acumatica naming conventions </value>
   </data>
+  <data name="PX1110Fix" xml:space="preserve">
```

Comment:
@EAndrosova please review the messages for the new PX1110 diagnostic and code fix